src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingResponseMapper.java

```diff
@@ -203,52 +209,53 @@ private OrganisationDetails setOrganisationDetails(HearingPartyEntity hearingPar
     }
 
 
-    private ArrayList<HearingResponse> setHearingResponse(HearingEntity hearingEntity) {
-        ArrayList<HearingResponse> hearingResponses = new ArrayList<>();
-        for (HearingResponseEntity hearingResponseEntity : hearingEntity.getHearingResponses()) {
-            HearingResponse hearingResponse = new HearingResponse();
+    private HearingResponse setHearingResponse(HearingEntity hearingEntity) {
```

Comment:
As per latest spec with audit changes, we should only be returning one rather than a list like previously.